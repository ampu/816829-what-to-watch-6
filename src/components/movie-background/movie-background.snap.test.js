import React from 'react';

import {render} from '@testing-library/react';

import MovieBackground from './movie-background';

describe(`MovieBackground component`, () => {
  it(`Should render according to the snapshot with no movie poster provided`, () => {
    const {container} = render(<MovieBackground/>);
    expect(container).toMatchSnapshot();
  });

  it(`Should render according to the snapshot with movie background provided`, () => {
    const movie = {
      title: `fakeTitle`,
      background: `fakeBackground`,
    };
    const {container} = render(<MovieBackground movie={movie}/>);
    expect(container).toMatchSnapshot();
  });
});
