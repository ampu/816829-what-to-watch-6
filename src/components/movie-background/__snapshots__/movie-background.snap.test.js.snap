// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieBackground component Should render according to the snapshot with movie background provided 1`] = `
<div>
  <div
    class="movie-card__bg"
  >
    <img
      alt="fakeTitle"
      src="fakeBackground"
    />
  </div>
</div>
`;

exports[`MovieBackground component Should render according to the snapshot with no movie poster provided 1`] = `
<div>
  
</div>
`;
